docker/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
   yorkie:
     volumes:
     - ./yorkie.json:/app/yorkie.json
-    image: 'yorkieteam/yorkie:latest'
+    image: 'yorkieteam/yorkie:0.1.0'
```

Comment:
We recently released a new version. How about using v0.1.1?

- https://hub.docker.com/repository/docker/yorkieteam/yorkie/tags
- https://www.npmjs.com/package/yorkie-js-sdk

Reply:
@hackerwins There seem to be a bug in using 0.1.1 

The cause has not been determined yet, so I fixed it at 0.1.0

Or Would I find the cause and create a PR that fixes 0.1.1?

Reply:
I didn't know there was a bug in v0.1.1. 🤔 First use v0.1.0 and would you like to report the bug?